test/pycheck/utils.py

```diff
@@ -258,7 +270,18 @@ def __getattr__(self, name):
 AsyncConnection = psycopg.AsyncConnection
 
 
-class Cursor:
+class OutputSilencer:
+    @contextmanager
+    def silent_logs(self):
+        """Set the log level to FATAL, so that we don't get any logs"""
+        buffer = io.StringIO()
+        with redirect_stdout(buffer):
+            self.sql("SET log_min_messages TO FATAL")
+            yield
+            self.sql("RESET log_min_messages")
```

Comment:
I think it would be good to clean up using a `finally`.
(don't apply suggestion verbatim it mixes tabs and spaces it seems)
```suggestion
			try:
            	yield
            finally:
            	self.sql("RESET log_min_messages")
```